pkg/volume/rbd/attacher.go

```diff
@@ -214,6 +215,18 @@ func (detacher *rbdDetacher) UnmountDevice(deviceMountPath string) error {
 	}
 	klog.V(3).Infof("rbd: successfully umount device mountpath %s", deviceMountPath)
 
+	if devicePath == "" {
+		deviceMountedPathSeps := strings.Split(deviceMountPath, "-image-")
+		if len(deviceMountedPathSeps) != 2 {
+			return fmt.Errorf("Can't found devicePath for %v", deviceMountPath)
+		}
```

Comment:
@knightXun could you write a utility function to extract pool and image name from device mount path in rbd_util.go alongside makePDNameInternal, and write some unit tests？
Current split logic seems not right, it is an absolute path.